src/material/menu/menu-item.ts

```diff
@@ -47,6 +47,7 @@ const _MatMenuItemMixinBase: CanDisableRippleCtor & CanDisableCtor & typeof MatM
     '[class.mat-menu-item]': 'true',
     '[class.mat-menu-item-highlighted]': '_highlighted',
     '[class.mat-menu-item-submenu-trigger]': '_triggersSubmenu',
+    '[class.mat-focus-indicator]': 'true',
```

Comment:
Prefer `'class': 'mat-focus-indicator'` or `'[class.mat-focus-indicator]': 'true'`? I've seen both used.

Reply:
We only use the latter in certain edge cases (like a child class needing to _not_ have that class)